### ISSUE TITLE
Cursor vanishes after sauciying

### ISSUE THREAD

**Issue by Calyhre** [muxumuxu/saucisse-cli#2]
When I want to move my body, and I want to do it so many times every days, I used to launch `saucisse`. 
But then, my cursor in my shell is missing when I stop sauciying :(

My terminal is TotalTerminal v1.5.4, and I've powered all the things up with ZSH

**Comment by caedes** [comment]
:+1:

**Comment by caedes** [comment]
I found this in the [README](https://github.com/chjj/blessed#example):

``` javascript
// Quit on Escape, q, or Control-C.
screen.key(['escape', 'q', 'C-c'], function(ch, key) {
  return process.exit(0);
});
```

**Comment by ArnaudRinquin** [comment]
Fixed in 4ab190f55e1fe0d3661933cc4e914e0b64b261ca
